### ISSUE TITLE
Terrain Finder doesnt work on underwater formations

### ISSUE THREAD

**Issue by is-ege** [coolmann24/TerrainFinderCpp#2]
I tried to find underwater formations and it failed, to make sure the problem wasnt on my end, I booted up random world on single player and tried several underwater terrain and it didnt work, The only false positive was a small island with the same terrain.

On your [video](https://www.youtube.com/watch?v=Trocrqd038o) you said the tool searches the stone noise map and places air automatically above blocks so that might be the problem.

This tool is amazing and I hope you fix this.

_(unrelated question: is it possible to make a tool that finds coords from placements of flowers, trees and other decorative elements and can you make it?)_

**Comment by TaranKamireddy** [comment]
Yes that is a problem and in the older version an option to toggle the air was available so hopefully, he will add that back to the new version.

**Comment by FullSendMC** [comment]
I need it to work underwater in a swamp.
